app/views/layouts/application.html.erb

```diff
@@ -32,6 +32,7 @@
     <%= render partial: "shared/header" %>
 
     <div class="govuk-width-container ">
+      <%= yield :breadcrumbs %>
```

Comment:
Is this a breadcrumb? https://design-system.service.gov.uk/components/back-link/